apps/ejabberd/src/mod_time.erl

```diff
@@ -0,0 +1,58 @@
+%%%-------------------------------------------------------------------
+%%% @author Ludwik Bukowski <ludwik.bukowski@erlang-solutions.com>
+%%% @copyright (C) 2015, Ludwik Bukowski
+%%% @doc XEP-0202: Entity Time
+%%% @end
+%%%-------------------------------------------------------------------
+-module(mod_time).
+-author('ludwik.bukowski@erlang-solutions.com').
+-behaviour(gen_mod).
+-export([start/2, stop/1, process_local_iq/3]).
+-include("ejabberd.hrl").
+-include("jlib.hrl").
+
+start(Host, Opts) ->
+    mod_disco:register_feature(Host, ?NS_TIME),
+    IQDisc = gen_mod:get_opt(iqdisc, Opts,
+                             one_queue),
+    gen_iq_handler:add_iq_handler(ejabberd_local, Host,
+                                  ?NS_TIME, ?MODULE, process_local_iq, IQDisc).
+
+
+stop(Host) ->
+    mod_disco:unregister_feature(Host, ?NS_TIME),
+    gen_iq_handler:remove_iq_handler(ejabberd_local, Host,
+                                     ?NS_TIME).
+
+
+process_local_iq(_From, _To, #iq{type = set, sub_el = SubEl} = IQ) ->
+    IQ#iq{type = error, sub_el = [SubEl, ?ERR_NOT_ALLOWED]};
+
+process_local_iq(_From, _To, #iq{type = get} = IQ) ->
+    Now = now(),
+    Now_universal = calendar:now_to_universal_time(Now),
+    Now_local = calendar:now_to_local_time(Now),
+    {UTC_time, UTC_diff} = jlib:timestamp_to_iso(Now_universal, utc),
+    UTC = list_to_binary(UTC_time ++ UTC_diff),
+    Seconds_diff =
+    calendar:datetime_to_gregorian_seconds(Now_local) -
```

Comment:
Actually I meant this:
```erlang
Seconds_diff =
calendar:datetime_to_gregorian_seconds(Now_local) -
calendar:datetime_to_gregorian_seconds(Now_universal),
```

What you did is a good change but I still what to see above 3 lines done more readably  :)